bitnami/valkey/templates/scripts-configmap.yaml

```diff
@@ -74,39 +74,39 @@ data:
 
     HEADLESS_SERVICE="{{ template "common.names.fullname" . }}-headless.{{ include "common.names.namespace" . }}.svc.{{ .Values.clusterDomain }}"
 
-    if [ -n "$VALKEY_EXTERNAL_MASTER_HOST" ]; then
-        VALKEY_SERVICE="$VALKEY_EXTERNAL_MASTER_HOST"
+    if [ -n "$VALKEY_EXTERNAL_PRIMARY_HOST" ]; then
+        VALKEY_SERVICE="$VALKEY_EXTERNAL_PRIMARY_HOST"
     else
         VALKEY_SERVICE="{{ template "common.names.fullname" . }}.{{ include "common.names.namespace" . }}.svc.{{ .Values.clusterDomain }}"
     fi
 
     SENTINEL_SERVICE_PORT=$(get_port "{{ include "common.names.fullname" . }}" "SENTINEL")
     validate_quorum() {
         if is_boolean_yes "$VALKEY_TLS_ENABLED"; then
-            quorum_info_command="{{- if and .Values.auth.enabled .Values.auth.sentinel }}REDISCLI_AUTH="\$VALKEY_PASSWORD" {{ end }}valkey-cli -h $VALKEY_SERVICE -p $SENTINEL_SERVICE_PORT --tls --cert ${VALKEY_TLS_CERT_FILE} --key ${VALKEY_TLS_KEY_FILE} --cacert ${VALKEY_TLS_CA_FILE} sentinel master {{ .Values.sentinel.masterSet }}"
+            quorum_info_command="{{- if and .Values.auth.enabled .Values.auth.sentinel }}REDISCLI_AUTH="\$VALKEY_PASSWORD" {{ end }}valkey-cli -h $VALKEY_SERVICE -p $SENTINEL_SERVICE_PORT --tls --cert ${VALKEY_TLS_CERT_FILE} --key ${VALKEY_TLS_KEY_FILE} --cacert ${VALKEY_TLS_CA_FILE} sentinel primary {{ .Values.sentinel.primarySet }}"
```

Comment:
Can we confirm the "sentinel primary" command exists in the `valkey-cli`?

Reply:
[Link](https://github.com/valkey-io/valkey/releases/tag/8.0.0-rc2#:~:text=Add%204%20sentinel%20command%20GET%2DPRIMARY%2DADDR%2DBY%2DNAME%2C%20PRIMARY%2C%20PRIMARIES%2C%20and%0AIS%2DPRIMARY%2DDOWN%2DBY%2DADDR%20to%20allow%20clients%20to%20use%20inclusive%20language.)

Reply:
Also tested:

```
$ REDISCLI_AUTH="$VALKEY_PASSWORD" valkey-cli -h valkey -p 26379 sentinel primary myprimary
 1) "name"
 2) "myprimary"
 3) "ip"
 4) "valkey-node-0.valkey-headless.test.svc.cluster.local"
 5) "port"
 6) "6380"
...
```